.github/workflows/testing.yml

```diff
@@ -11,6 +11,7 @@ jobs:
         uses: actions/checkout@v2.3.4
       - name: Install dependencies
         run: |
+          sudo apt-get update --fix-missing
```

Comment:
What's this necessary for?   This shouldn't be necessary since everything is done from a clean install and any dependency should be explicitly listed.

Reply:
the --fix-missing is indeed unnecessary, took it out. The update seems to be needed: the apt-get install has failed in the last few test runs because it is unable to find packages, seems to be running off some older list?